### PR TITLE
Use warning if pipeline superseded

### PR DESCRIPTION
**Pull request by vsramalwan**
If the the Codepipeline is superseded it returns an error in GHA. This PR will return an warning rather an error and the GHA won't fail.

### CODE REVIEW

File: src/index.ts [hunk 1]
```diff
@@ -55,8 +55,8 @@ const waitForPipeline = async (pipelineName: string, pipelineExecutionId: string
         core.error(`Pipeline '${pipelineName}' stopped.`);
         return false;
       case PipelineExecutionStatus.Superseded:
-        core.error(`Pipeline '${pipelineName}' was superseded.`);
-        return false;
+        core.warning(`Pipeline '${pipelineName}' was superseded. Skipping rest of the execution.`);
+        return true;
```
Comment (vsramalwan) [inline src/index.ts:59]: @snowiow this might still show Green in the GHA itself after throwing a warning. Not sure, how can we show skip sign there 🤔

Comment (snowiow) [reply src/index.ts:59]: I think thats not possible because the action already ran and the skip sign is only shown if the action did not run at all

**Review by snowiow — APPROVED** [review]

**Comment by snowiow** [comment]
Can you update the README with v0.2.2 in the examples?

**Comment by snowiow** [comment]
ah yes and currently the js still needs to be build manually 😄 `npm run build`

**Review by snowiow — APPROVED** [review]
🙏

**Pull request merged**